src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/code/CodeCache.java

```diff
@@ -92,12 +92,14 @@ public CodeBlob findBlobUnsafe(Address start) {
     return createCodeBlobWrapper(cbAddr, start);
   }
 
-  public CodeBlob createCodeBlobWrapper(Address cbAddr, Address start) {
+  // cbAddr - address of a code blob
+  // cbPC   - address inside of a code blob
+  public CodeBlob createCodeBlobWrapper(Address cbAddr, Address cbPC) {
```

Comment:
Can you change findBlobUnsafe() above also? That's where the naming problem originated.